### PR TITLE
Maxwellfundi covertcols

### PR DESCRIPTION
**Pull request by dannyparsons**

### CODE REVIEW

File: instat/dlgConvertColumns.vb [hunk 1]
```diff
@@ -64,136 +54,78 @@ Public Class dlgConvertColumns
...
+        ucrChkSpecifyDecimalsToDisplay.SetText("Specify Decimals (from Numeric)")
+        ucrChkSpecifyDecimalsToDisplay.SetParameter(New RParameter("set_digits"))
+        ucrChkSpecifyDecimalsToDisplay.SetValuesCheckedAndUnchecked("TRUE", "FALSE")
+        ucrChkSpecifyDecimalsToDisplay.SetRDefault("FALSE")
+        ucrChkSpecifyDecimalsToDisplay.AddToLinkedControls(ucrLinked:=ucrNudDisplayDecimals, objValues:={True}, bNewLinkedAddRemoveParameter:=True, bNewLinkedHideIfParameterMissing:=True)
+
+        ucrNudDisplayDecimals.SetParameter(New RParameter("set_decimals"))
+        ucrNudDisplayDecimals.Minimum = 1
+        ucrNudDisplayDecimals.Increment = 1
+        ucrNudDisplayDecimals.SetRDefault("4")
```
Comment (dannyparsons) [inline instat/dlgConvertColumns.vb]: @maxwellfundi See here how I have corrected the linking.

**Comment by dannyparsons** [comment]
@africanmathsinitiative/developers Convert Columns dialog ready for review

File: instat/dlgConvertColumns.vb (hunk 1 above)
Comment (maxwellfundi) [inline instat/dlgConvertColumns.vb]: Great, i will definitely go through it.

**Comment by dannyparsons** [comment]
Sorry, deleted this accidentally, still ready for review.

File: instat/dlgConvertColumns.vb [hunk 2]
```diff
@@ -64,136 +54,78 @@ Public Class dlgConvertColumns
...
 
-    Private Sub ucrReceiverColumnsToConvert_SelectionChanged() Handles ucrReceiverColumnsToConvert.SelectionChanged
-        If Not ucrReceiverColumnsToConvert.IsEmpty Then
-            ucrBase.clsRsyntax.AddParameter("col_names", ucrReceiverColumnsToConvert.GetVariableNames)
-        Else
-            ucrBase.clsRsyntax.RemoveParameter("col_names")
-        End If
-        TestOKEnabled()
-    End Sub
+        ucrPnlConvertTo.SetParameter(New RParameter("to_type"))
```
Comment (lilyclements) [inline instat/dlgConvertColumns.vb]: You need to number your parameters

Comment (dannyparsons) [inline instat/dlgConvertColumns.vb]: Thanks, corrected.

**Pull request merged**